app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -245,7 +243,7 @@ def server_status
     def facts
       User.as_anonymous_admin do
         sync_task(::Actions::Katello::Host::Update, @host, rhsm_params)
-        @host.subscription_facet.update_facts(rhsm_params[:facts]) unless rhsm_params[:facts].blank?
+        Katello::Host::SubscriptionFacet.update_facts(@host, rhsm_params[:facts])
```

Comment:
I don't see why we'd call a class method to then call methods on the object instead of having an instance method on `subscription_facet` like the original line had?